filters/redirect/redirect.go

```diff
@@ -43,7 +44,21 @@ func (f *Redirect) Request(ctx skipper.FilterContext) {}
 
 func (f *Redirect) Response(ctx skipper.FilterContext) {
 	w := ctx.ResponseWriter()
-	w.Header().Set("Location", f.location)
+
+	u, err := url.Parse(f.location)
+
+	if err != nil {
+		w.WriteHeader(501)
```

Comment:
So, this is supposed to be a scenario of the redirect filter which contained an invalid destination URL? If it was, why wasn't it parsed at load time and rejected if route was invalid? Arguably, Inkeeper could've rejected it also...